src/main/java/org/springframework/integration/aws/support/KplBackpressureException.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017-2024 the original author or authors.
```

Comment:
This class was not created in `2017`.
Or better to say: committed to code base.
So, I prefer to start from the actual year when it is committed.
Therefore in our case, since today is the last day of 2024, I'd suggest to change this copyright just to `2025` or `2025-2025`. Just because we might not merge this today.
And, please, change the copyright in other affected classes.